### ISSUE TITLE
(Feature request) Add support for opus/mp3/ogg playback (bug?)

### ISSUE THREAD

**Issue by hardBSDk** [oxen-io/session-android#777]
For some reason I can't play some file types, I press to play and it does nothing, I think there's mature open source libraries for these formats.

**Comment by hjubb** [comment]
Are you aware if the senders of those files are from a different client like desktop? I'll look into increasing support for more audio codecs if we don't already support them :+1:

**Comment by hardBSDk** [comment]
> Are you aware if the senders of those files are from a different client like desktop? I'll look into increasing support for more audio codecs if we don't already support them +1

No, this file types are stored in my Android, I sent them and tried to play.

Android version: 9
Session version: 1.11.11
